tests/virtualization/xen/virtmanager_add_devices.pm

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: This test adds some devices to our VMs
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base "x11test";
+use xen;
+use strict;
+use testapi;
+use utils;
+use virtmanager 'detect_login_screen';
+
+sub run {
+    my ($self) = @_;
```

Comment:
why do you have `my ($self) = @_;` in most modules? - asking because I see it more often lately and variable is defined but not used

Reply:
I will try to avoid those in my next PR :smile: